auth-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+info:
+  component: Authentication Service
+
+spring:
+  application:
+     name: AUTH-SERVICE
+  freemarker:
+    enabled: false
+  thymeleaf:
+    cache: false
+
+server:
+  context-path: /uaa
+  port: 8085
```

Comment:
Could you please add the 'Allocated ports' table to the our POC document and fill the first record something like AUTH-SERVER | 8085